test/export/StructureDefinitionExporter.test.ts

```diff
@@ -95,4 +96,63 @@ describe('StructureDefinitionExporter', () => {
       exporter.exportStructDef(extension, input);
     }).toThrow('Parent Bar not found for Foo');
   });
+
+  // Rules
+  it('should emit an error and continue when the path is not found', () => {
+    // TODO: This should check for emitting an error once we have logging
+    const profile = new Profile('Foo');
+    const rule = new CardRule('fakePath');
+    rule.min = 0;
+    rule.max = '1';
+    profile.rules.push(rule);
+    const structDef = exporter.exportStructDef(profile, input);
+    expect(structDef).toBeDefined();
+    expect(structDef.type).toBe('Resource');
+  });
+
+  // Card Rule
+  it('should apply a correct card rule', () => {
+    const baseProfile = new Profile('Foo');
+    baseProfile.parent = 'Observation';
+    const changedProfile = new Profile('Foo');
+    changedProfile.parent = 'Observation';
+
+    const rule = new CardRule('subject');
+    rule.min = 1;
+    rule.max = '1';
+    changedProfile.rules.push(rule);
+
+    const baseStructDef = exporter.exportStructDef(baseProfile, input);
```

Comment:
While this approach to getting the base cardinality is theoretically sound -- I think it might be better to get the base structure definition from the `FHIRDefinitions` and then load it via `StructureDefinition.fromJSON(...)`.  That way you really are dealing w/ the base resource and not a profile on it.

Maybe we should use this as an opportunity to implement `StructureDefinition.getBaseStructureDefinition()` -- which might be useful in other cases too.  Then you'd do:

```ts
const changedStructDef = exporter.exportStructDef(changedProfile, input);
const baseStructDef = changedStructDef.getBaseStructureDefinition();
```

Reply:
Yeah that makes sense. I think I could actually also use the `ElementDefinition` functions that tracks `diff`s. Could it be cleaner to use those here?

Reply:
I don't know...  If you use `diff`, I'd suggest you still check the regular `elements` _and_ the `diff`.  But really, I think we should have separate unit tests to ensure diff is working correctly anyway.  So I'd lean toward not using `diff` here.

If you don't want to pull and check the base definition here, I think you could also check the element's `base.min` and `base.max` which are also supposed to represent the original cardinality.  In past versions of FHIR though, I feel like `base` was not always present.

Reply:
Upon further investigation, I don't think `diff` is used yet when we make a new `ElementDefinition`, so I will go the `getBaseStructureDefinition` route.